src/constants.ts

```diff
@@ -83,9 +83,21 @@ export const CLASSES = {
 
 // 1-based in case we ever do `if(element.fontFamily)`
 export const FONT_FAMILY = {
-  Virgil: 1,
-  Helvetica: 2,
-  Cascadia: 3,
+  Virgil: {
+    id: 1,
+    fallback:
+      "'Comic Sans MS', 'Segoe Print', 'Bradley Hand', 'Lucida Handwriting', 'Marker Felt', cursive",
+  },
+  Helvetica: {
+    id: 2,
+    fallback:
+      "-apple-system,BlinkMacSystemFont, 'Segoe UI', 'Noto Sans', Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji'",
+  },
+  Cascadia: {
+    id: 3,
+    fallback:
+      "ui-monospace, SFMono-Regular, 'SF Mono', Menlo, Consolas, 'Liberation Mono', monospace",
+  },
```

Comment:
let's rather define a separate map with fallbacks to not break the API, especially since it's not 100% clear that this should ultimately be the solution for fixing the font loading issue.

Reply:
ok no problem, I'll do a map like this:

```js
{
  Virgil: "'Comic Sans MS', ...",
  // ...
}
```

Reply:
since this + imports was most of the change, maybe it makes more sense to just restart this PR from scratch 🤔

Reply:
Opened new PR:

- https://github.com/excalidraw/excalidraw/pull/6550